src/solvers/changeCobraSolverParams.m

```diff
@@ -56,9 +56,9 @@
 allowedQPparams = {'minNorm', 'printLevel'};
 allowedMILPparams = {'intTol','relMipGapTol','timeLimit','logFile','printLevel'};
 
-% Only LP is currently included
+% Only LP, QP and MILP are currently included

```

Comment:
At least from my testing with `changeCobraSolver`, the global variable setting worked, but let me test this again.